tests/snuba/api/endpoints/test_organization_events_stats_mep.py

```diff
@@ -1458,3 +1458,33 @@ def test_group_by_transaction(self):
             [{"count": 5.0}],
             [{"count": 10.0}],
         ]
+
+    def test_on_demand_table_user_misery(self):
+        resp = self._on_demand_query_check(
+            {
+                "dataset": "metricsEnhanced",
+                "environment": "prod",
+                "interval": "30m",
+                "onDemandType": "dynamic_query",
+                "orderby": "",
+                "partial": 1,
+                "project": self.project.id,
+                "query": "",
+                "referrer": "api.dashboards.widget.line-chart",
+                "statsPeriod": "7d",
+                "useOnDemandMetrics": "true",
+                "yAxis": "user_misery(300)",
+            },
+        )
+        for datum in resp.data["data"]:
+            # XXX: Find a way to have some user misery data
+            assert datum[1] == [{"count": 0}]
```

Comment:
I'm working now to make sure the test returns data rather than everything being prefilled with zeros.
In production, [this API call](https://us.sentry.io/api/0/organizations/sentry/events-stats/?dataset=metricsEnhanced&environment=prod&interval=30m&onDemandType=dynamic_query&orderby=&partial=1&project=1&query=&referrer=api.dashboards.widget.line-chart&statsPeriod=7d&useOnDemandMetrics=true&yAxis=user_misery%28300%29) does return data. It's just here in our tests that I may be doing something wrong.
<img width="340" alt="image" src="https://github.com/getsentry/sentry/assets/44410/2aad365e-8f7f-4f20-9f10-0d501a03f89d">